src/LuaPiGui.cpp

```diff
@@ -0,0 +1,1092 @@
+// Copyright © 2008-2016 Pioneer Developers. See AUTHORS.txt for details
+// Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+#include "LuaPiGui.h"
+#include "LuaUtils.h"
+#include "LuaConstants.h"
+#include "PiGui.h"
+#include "WorldView.h"
+#include "Pi.h"
+#include "Game.h"
+#include "graphics/Graphics.h"
+#include "Player.h"
+
+#undef RegisterClass
```

Comment:
That's because on Windows [RegisterClass](https://msdn.microsoft.com/en-us/library/windows/desktop/ms633586(v=vs.85).aspx) is defined as something else. It's `#undef` there to avoid dragging in another header where it's undef'd elsewhere.